repos/fdbt-site/tests/pages/api/serviceList.test.ts

```diff
@@ -1,107 +1,223 @@
 import {
     expectedPeriodMultipleServicesTicketWithMultipleProducts,
     getMockRequestAndResponse,
+    zoneStops,
 } from '../../testData/mockData';
 import serviceList from '../../../src/pages/api/serviceList';
 import {
     FARE_TYPE_ATTRIBUTE,
     MATCHING_JSON_ATTRIBUTE,
     MATCHING_JSON_META_DATA_ATTRIBUTE,
+    SERVICE_LIST_ATTRIBUTE,
+    STOPS_EXEMPTION_ATTRIBUTE,
 } from '../../../src/constants/attributes';
 import * as userData from '../../../src/utils/apiUtils/userData';
+import * as fileUpload from '../../../src/utils/apiUtils/fileUpload';
+import * as sessions from '../../../src/utils/sessions';
+import * as virusCheck from '../../../src/utils/apiUtils/virusScan';
+import * as auroradb from '../../../src/data/auroradb';
+import { secondTestCsv } from '../../testData/csvZoneData';
```

Comment:
could we give this a more relevant name?

Reply:
We could change it yeah, but then it wouldnt make sense in this context because we arent using it as a fare zone.